src/views/tickets/view-link.php

```diff
@@ -4,23 +4,21 @@
  * Included on the Events Single Page after the meta
  * The Message that will link to the Tickets page
  *
- * Override this template in your own theme by creating a file at [your-theme]/tribe-events/tickets/view-link.php
+ * Override this template in your own theme by creating a file at [your-theme]/tribe-events/tickets/orders-link.php
  *
  * @since 4.2
  * @since 4.10.8 Renamed template from order-links.php to view-link.php. Updated to not use the now-deprecated third
- *            parameter of `get_description_rsvp_ticket()`
+ *               parameter of `get_description_rsvp_ticket()`
  *
  * @version 4.10.8
```

Comment:
This view needs a `@since TBD __what changed__` and `@version TBD`

Reply:
done